docs/production-deployment/cloud/nexus/index.mdx

```diff
@@ -18,71 +18,368 @@ keywords:
-
-**Advantages of Temporal Nexus:**
-
-- Connect durable execution within and across Namespaces.
-- Service contracts suitable for sharing across teams.
-- Abstract underlying Temporal primitives, like Workflows – or execute arbitrary code.
-- Modular design that streamlines multi-team development.
-- Improved security and blast-radius isolation with isolated Namespaces.
-- Same queue-based Worker architecture, no bespoke service deployments needed.
+Nexus Services are exposed from a Nexus Endpoint that is created in the Nexus Registry.
+Nexus Endpoints include service documentation that enables others to use the Nexus Services provided by a Nexus Endpoint, for example from caller Workflows.
+
+![Nexus Overview](/img/nexus-overview.png)
+
+Temporal has built-in Nexus Machinery that performs the low-level [Nexus RPC](/glossary#nexus-rpc) operations on the wire and provides an integrated Temporal SDK experience to create Nexus Services in a handler Worker and use them from a caller Workflow through a Nexus Endpoint.
+
+In Temporal Cloud, the Nexus Machinery routes Nexus requests across Namespace boundaries through a secure mTLS Envoy mesh, where they are sync matched to a Worker in the handler Namespace, which processes the Nexus request.
+
+Nexus caller Workflows will send commands to Temporal Cloud to schedule or cancel a Nexus Operation in the caller Namespace.
+Handler Workers will poll for Nexus tasks in the handler Namespace.
+
+Nexus Operation handlers can be implemented as [Asynchronous Operations](/nexus/services-operations#asynchronous-operation-lifecycle) or [Synchronous Operations](/nexus/services-operations#asynchronous-operation-lifecycle), using Temporal SDK builders that enable running a Nexus Service in a Worker, typically the same Worker as the underlying primitives the Nexus Service abstracts.
+
+For example, an asynchronous operation that starts a Workflow in a different Namespace:
+
+Unlike a traditional RPC, an arbitrary-duration Nexus Operation has an identity (ID) that lasts beyond the process lifetimes of the caller and handler, and can be used to re-attach to a long-lived Nexus Operation, for example one backed by a Temporal Workflow.
+
+When you schedule a Nexus Operation in a caller Workflow, a command is sent to Temporal to schedule the Operation, and then the Temporal Nexus Machinery is responsible for making the Nexus RPC calls on your behalf.
+This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with a Temporal Service.
+
+![NexusYourCloud](/img/nexus/NexusYourCloud.png)
+
+Similar to executing an Activity, to execute a Nexus Operation from a Workflow:
```

Comment:
I don't think you need to point out correspondences of previous "how it works" here. Also, consider going through these steps, and talking them out loud as if you were giving instructions and explanations in person. Sometimes this helps to make the writing more fluent. "A request starts with a Workflow calling to schedule a Nexus Operation. This sends a `SNO` command using RPC.  ~~The command is sent as part of `RWTC` and~~ the Workflow Execution history records `NOS`. The Nexus request is now ready to be worked on. ... Next, a Nexus-enabled Worker that can handle this request, polls the Nexus Task Queue and finds the request. It begins to process it. 

I know I'm blathering a little, but I'm trying to get across a  sense of storytelling in a way that focuses less on details, which can be summarized in a table, and more about the initiation and hand-off of responsibilities, etc.

Reply:
fixed